src/style.css

```
@@ -1 +1,14 @@
-/* styles go here */
+.fieldPersonalInfo,
+.fieldRegistration,
+.fieldFacts,
+.fieldAddInfo {
```

No needs to add o many different classes. Use generic one. LIke `"fieldset"`
```suggestion
 {
```